CI failure in unit tests:

```
>       self.assertEqual(result['msg'], 'dampening_routemap cannot be used with the dampening_half_time param')
E       AssertionError: 'parameters are mutually exclusive: dampening_routemap, dampening_half_time' != 'dampening_routemap cannot be used with the dampening_half_time param'
E       - parameters are mutually exclusive: dampening_routemap, dampening_half_time
E       + dampening_routemap cannot be used with the dampening_half_time param

test/units/modules/network/nxos/test_nxos_bgp_af.py:92: AssertionError
```

--- a/test/units/modules/network/nxos/test_nxos_bgp_af.py
+++ b/test/units/modules/network/nxos/test_nxos_bgp_af.py
@@ -89,7 +89,7 @@ class TestNxosBgpAfModule(TestNxosModule):
                              dampening_half_time=5, dampening_suppress_time=2000,
                              dampening_reuse_time=1900, dampening_max_suppress_time=10))
         result = self.execute_module(failed=True)
-        self.assertEqual(result['msg'], 'dampening_routemap cannot be used with the dampening_half_time param')
+        self.assertEqual(result['msg'], 'parameters are mutually exclusive: dampening_routemap, dampening_half_time')
 
     def test_nxos_bgp_af_client(self):
         set_module_args(dict(asn=65535, afi='ipv4', safi='unicast',

